src/cloud_provider/aws/kubernetes/mod.rs

```diff
@@ -243,6 +263,25 @@ impl<'a> EKS<'a> {
 
         let format_ips =
             |ips: &Vec<String>| -> Vec<String> { ips.iter().map(|ip| format!("\"{}\"", ip)).collect::<Vec<_>>() };
+        let format_zones = |zones: &Vec<AwsZones>| -> Vec<String> {
```

Comment:
Why a clojure/lambda ? You are invoking it only once

Reply:
To make it more clear to read